src/styles/main.scss

```diff
@@ -0,0 +1,117 @@
+$mainTxtColor: #060b35;
+$secondaryTxtColor: #616070;
+$cardMainCol: #fff;
+$cardBorder: #f3f3f3;
+$cardBtnColor: #00acdc;
+$gap: 16px;
+$mainFont: "Roboto", sans-serif;
+
+@mixin base($color, $borderCol) {
+  background-color: $color;
+  border: 1px solid $borderCol;
+  border-radius: 5px;
+}
```

Comment:
I think you're using this function only once, so there is not point of declaring function - it's better to keep it as simple as possible

Reply:
this one is actually used twice, once for card itself, second for button :)